php/sunrise-router-annotations/composer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "require": {
+        "sunrise/http-router": "^2.4",
+        "sunrise/http-factory": "^1.0"
```

Comment:
could you use  `~1.0.0`, to ensure in **1.0**

Reply:
mission complete